sdk/lib/rtl/priv.c

```diff
@@ -486,3 +486,78 @@ RtlAdjustPrivilege(IN ULONG Privilege,
 
     return STATUS_SUCCESS;
 }
+
+/*
+ * @implemented
+ */
+NTSTATUS
+NTAPI
+RtlRemovePrivileges(IN HANDLE TokenHandle,
+                    IN PULONG PrivilegesToKeep,
+                    IN ULONG PrivilegeCount)
+{
+    DPRINT("RtlRemovePrivileges(%p, %p, %u)\n", TokenHandle, PrivilegesToKeep, PrivilegeCount);
+
```

Comment:
This RTL function exported from ntdll.dll in user-mode, in this case, PAGED_CODE_RTL() is not necessary?

Reply:
There can be NTDLL functions that can be used in kernel mode on-demand. Is this `RtlRemovePrivileges` strictly used for user mode system components or can it have its usage in kernel mode too? If the former then you might not need `PAGED_CODE_RTL()`, otherwise you have to use it.

Reply:
Yes, `RtlRemovePrivileges` is exported in user-mode only (in Windows), not in kernel-mode, like `RtlAdjustPrivilege` but new for NT 6.0+

Reply:
@GeoB99 : Normally the RTL should be compiled twice: once for the kernel (making it use possible kernel-specific defines), and once for user-mode (for ntdll). We don't do that yet, and as a result we may end up (in non-optimized builds) with user-mode-only functions (like this one) being added in the kernel but being never used.